modules/internal/ChapelLocale.chpl

```diff
@@ -105,27 +105,36 @@ module ChapelLocale {
   enum localeKind { regular, any, nilLocale, dummy, default };
 
   pragma "locale private"
+  pragma "no doc"
   const nilLocale = new locale(localeKind.nilLocale);
   pragma "locale private"
+  pragma "no doc"
   var defaultLocale = new locale(localeKind.default);
 
   // dummyLocale is not locale private. We use it before locales initialized in
   // the first place, so it should stay in the locale that started the
   // execution.
+  pragma "no doc"
   var dummyLocale = new locale(localeKind.dummy);
 
   pragma "always RVF"
   record _locale {
+
+    pragma "no doc"
     var _instance: unmanaged BaseLocale?;
 
+    pragma "no doc"
     inline proc _value {
       return _instance!;
     }
 
     forwarding _value;
 
     // default initializer for the locale record.
-    // TODO: What is the default value for a locale?
+    /*
+      Default initializer for locale. The default value for a ``locale``
+      variable is ``Locales[0]``
+     */
```

Comment:
I'd nodoc all initializers and deinitializers for locales.  In fact, I'd be tempted to "no doc" the entire locale record and then move its methods out into `proc locale.foo()` style secondary method declarations so that the documentation can focus less on the locale type and more on methods that can be applied to the locale type (?).

(I did not think to try this in my own attempt to improve the locale module's documentation).